esmvalcore/cmor/_fixes/cmip5/miroc_esm.py

```diff
@@ -91,4 +94,18 @@ def fix_metadata(self, cubes):
             except CoordinateNotFoundError:
                 pass
 
+            # Fix time
+            if cube.coords('time'):
+                expected_time_units = Unit('days since 1950-1-1 00:00:00',
+                                           calendar='gregorian')
+                if cube.coord('time').units != expected_time_units:
+                    continue
+                if cube.coord('time').bounds is None:
+                    continue
+                if np.isclose(cube.coord('time').bounds[0][0], -711860.5):
```

Comment:
could you maybe write a comment above the if statement explaining what -711860.5 means? Future Manu in 5 years will be scratching his head :grin: